config/suit/templates/nrf54h20/default/v1/app_envelope.yaml.jinja2

```diff
@@ -0,0 +1,133 @@
+    # This increases the robustness against bit flips on the transport,
+    # for example when storing the data on an external memory device.
+    # The suit-directive-try-each will complete on the first successful subsequence.
+    - suit-directive-try-each:
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+    suit-invoke:
+    - suit-directive-set-component-index: 0
+    - suit-directive-invoke:
+      - suit-send-record-failure
+    suit-install:
+    - suit-directive-set-component-index: 1
+    - suit-directive-override-parameters:
+        suit-parameter-uri: '#{{ application['name'] }}'
+        suit-parameter-image-digest:
+          suit-digest-algorithm-id: cose-alg-sha-256
+          suit-digest-bytes:
+            file: {{ application['binary'] }}
+    - suit-directive-fetch:
+      - suit-send-record-failure
+    - suit-directive-try-each:
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+      - - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+    - suit-directive-set-component-index: 0
+    - suit-directive-override-parameters:
+        suit-parameter-source-component: 1
+    # When copying the data it is worth to retry the sequence of
+    # suit-directive-copy and suit-condition-image-match at least once.
+    # If a bit flip occurs, it might be due to a transport issue, not
+    # a corrupted candidate image. In this case the bit flip is recoverable
+    # and it is worth retrying the operation.
+    # The suit-directive-try-each will complete on the first successful subsequence.
+    - suit-directive-try-each:
+      - - suit-directive-copy:
+          - suit-send-record-failure
+        - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+      - - suit-directive-copy:
+          - suit-send-record-failure
+        - suit-condition-image-match:
+          - suit-send-record-success
+          - suit-send-record-failure
+          - suit-send-sysinfo-success
+          - suit-send-sysinfo-failure
+    suit-text:
+      suit-digest-algorithm-id: cose-alg-sha-256
+    suit-manifest-component-id:
+    - INSTLD_MFST
+    - RFC4122_UUID:
+        namespace: {{ mpi_application_vendor_name }}
+        name: {{ mpi_application_class_name }}
+  suit-text:
```

Comment:
If possible, please add the support for `suit-candidate-verification` sequence.

Reply:
This manifest was moved from the `suit-generator` repository into `sdk-nrf` without changes. Could this be done in subsequent PRs?